### ISSUE TITLE
Android icons should be rounded

### ISSUE THREAD

**Issue by daniel** [RealFaviconGenerator/realfavicongenerator#180]
I think you should make the Android icons rounded. When I tried to generator I got square icons on Android home screen, so it doesn't seem they are automatically rounded on Android.

**Comment by phbernard** [comment]
Hi Daniel,

Are we talking about Android Chrome? Because the builtin Android browser tend to produce various (and poor...) results.

If we are talking about the Android Chrome icon, how did you design it? With rounded corners something else?

**Comment by daniel** [comment]
Yes, I'm using the latest Chrome browser (on a Google Nexus 5, Android 5.1.1).
I submitted a square picture to realfavicongenerator, in accordance with the instructions there.

**Comment by phbernard** [comment]
Oh, ok, I get it. When you select "Add a solid, plain background to fill the transparent regions", the preview shows an icon with rounded corners. But when you add the site to the home screen, the icon has no rounded corners. Unlike iOS, Android doesn't "round" the icon. So the preview conflicts with the actual result.

The fix you suggest is correct: Android icons should be rounded in this case.

**Comment by daniel** [comment]
I didn't select "Add a solid, plain background to fill the transparent regions", did you mean "Add margins and a plain background."? I kept the Main settings as "No change, keep the master picture as it is.". The icon is rounded in the preview for that option too.
If I understand correctly, realfavicongenerator never rounds icons when generating them for Android Chrome?
As a workaround, will it work if I upload a dedicated picture for Android Chrome that has rounded corners?

**Comment by phbernard** [comment]
Let's summarize:
- "No change, keep the master picture as it is.": The preview should be changed because is shows rounded corners whereas Android doesn't do this. The generated picture itself is correct.
- "Add margins and a plain background.": The preview is good, because rounded corners are a good on-size-fits-all setting. But the generated pictures are not correct, because they should have rounded corners.
- "Apply a slight drop shadow": Preview and actual result are consistent. No change here.

Does this sound good to you?

**Comment by daniel** [comment]
The text above the options says "Android is flexible regarding icon design. However, it automatically rounds icons", so that text should be changed too, to avoid confusion.
Shouldn't there be an option that just rounds the corners without adding margins or background?
Otherwise it sounds good.
